test/auth/oauth.tests.js

```diff
@@ -559,10 +559,11 @@ describe('OAuthAuthenticator', function() {
       this.authenticator.socialSignIn(userData, done.bind(null, null));
     });
 
-    it('should return a promise when no callback is given', function() {
-      var returnValue = this.authenticator.socialSignIn(userData);
-
-      expect(returnValue).to.be.an.instanceOf(Promise);
+    it('should return a promise when no callback is given', function(done) {
```

Comment:
Why did you change this? It's passing already for me locally. Do you mind to rollback this change please?

Reply:
I 
changed it because the test was wrong. It says that "it should return a promise", and a promise in Javascript is everything that implements the then/catch/finally interface. Previously, it required that it returned a **Bluebird** promise.

A few years ago that's all we had, but nowadays we have a native Promise implementation. Most libraries are updating the code to remove Bluebird promises, and so did **rest-facade**.

If I roll it back the tests won't pass. Right now the tests are passing because I rolled back several changes in **rest-facade** to prevent this bug (the authentication header issue) from breaking this auth0 library. 

**rest-facade** is updating soon and it won't be a major change since it doesn't change its public API. Therefore, if this test isn't updated, the tests will start failing.

Reply:
@ngonzalvez 
- do we need to update the minimum required version of rest-facade for this?
- are you saying that you replaced Bluebird promises for native promises without a breaking change?

I'm just trying to assess what's the risk of merging this. Thanks!